pod-ext/js/constants.js

```diff
@@ -0,0 +1,7 @@
+const constants = Object.freeze({
+	ORIGIN: "http://localhost:3001",
+	REMOTE: "POD_BROWSER",
+	HOST: "POD_EXT",
+	REGISTER: "REGISTER",
```

Comment:
So this one is defined as in the main `pod-app`'s [README](https://github.com/golemfactory/proofofdevice#native-messages-handled-by-pod-app) to have the JSON format:

`{ "msg" : "get_quote", "spid": "01234567abcdef" }`